.kickoff/tasks/watch.js

```diff
@@ -3,21 +3,28 @@
  */
 const config = require('../config');
 const gulp = require('gulp');
+const runSequence = require('run-sequence');
 
 gulp.task('watch', ['compile'], () => {
 	gulp.watch(['**/*.scss'], { cwd: config.css.scssDir }, ['css'])
```

Comment:
this here takes care of CSS copy part - https://github.com/TryKickoff/statix/blob/next/.kickoff/tasks/css.js#L71